docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -87,21 +125,27 @@ Create the device configuration file, named `my.custom.device.config.toml`, as s
   Labels = [ "MQTT", "test" ]
   [DeviceList.Protocols]
     [DeviceList.Protocols.mqtt]
+       # Comment out/remove below to use multi-level topics
        CommandTopic = "CommandTopic"
+       # Uncomment below to use multi-level topics
+       # CommandTopic = "command/my-custom-device"
     [[DeviceList.AutoEvents]]
        Interval = "30s"
        OnChange = false
        SourceName = "message"
 ```
 
-- `CommandTopic` is used to publish the GET or SET command request
+!!! note
+    `CommandTopic` is used to publish the GET or SET command request 
+    
+    **Using Multi-level Topic:** Remember to set `CommandTopic = "command/my-custom-device"`.
```

Comment:
No longer needed with moving the `Enabling Multi-Level Topics` section down below.